imports/ui/components/newTask/newTask.js

```diff
@@ -108,13 +112,17 @@ Template.newTask.events({
       //Urgent
       is_urgent = true;
     }
+    if(radioValue == "4") {
+      //Urgent
+      is_backlog = true;
```

Comment:
Am I correct in understanding that is_backlog = someday?
If so, what's the different terminology for? It had me confused for a little bit.